packages/dashboard/src/components/Card/examples/LineAndVerticalBarChartExample.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { ICardProps, CardSize, Priority, CardContentType } from '../Card.types';
+import { Card } from '../Card';
```

Comment:
Can you fix the imports to use something like 
`
import {
  Card,
  CardContentType,
  CardSize,
  ChartType,
  DashboardGridBreakpointLayouts,
  DashboardGridLayout,
  IAction,
  IThumbnailItemProps,
  Priority,
  Size
} from '@uifabric/dashboard';
`

Reply:
made import changes to previous examples as well.